tests/gnrc_udp/udp.c

```diff
@@ -138,6 +138,11 @@ static void send(char *addr_str, char *port_str, char *data_len_str, unsigned in
         if (iface > 0) {
             gnrc_pktsnip_t *netif = gnrc_netif_hdr_build(NULL, 0, NULL, 0);
 
+            if(netif == NULL) {
+                puts("Error: unable to allocate NETIF header");
+                gnrc_pktbuf_release(payload);
```

Comment:
Since `payload` is already part of the list `ip` you should release `ip` here (which in turn also releases `udp` and `payload`).